### ISSUE TITLE
NPE on server startup

### ISSUE THREAD

**Issue by SlimeDog** [lucko/LuckPerms#2051]
Spigot 1.15.2 (Jenkins 2645)
LuckPerms 5.0.91

```
[17:36:25] [Server thread/ERROR]: Error occurred while enabling LuckPerms v5.0.91 (Is it up to date?)
java.lang.NullPointerException: null
        at com.google.common.collect.ImmutableMap.copyOf(ImmutableMap.java:391) ~[spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at me.lucko.luckperms.common.query.QueryOptionsImpl.<init>(QueryOptionsImpl.java:58) ~[?:?]
        at me.lucko.luckperms.common.query.QueryOptionsBuilderImpl.build(QueryOptionsBuilderImpl.java:171) ~[?:?]
        at net.luckperms.api.query.DefaultQueryOptions.setup(DefaultQueryOptions.java:45) ~[?:?]
        at net.luckperms.api.query.DefaultQueryOptions.contextual(DefaultQueryOptions.java:50) ~[?:?]
        at net.luckperms.api.query.QueryOptions.defaultContextualOptions(QueryOptions.java:110) ~[?:?]
        at me.lucko.luckperms.common.query.QueryOptionsBuilderImpl.build(QueryOptionsBuilderImpl.java:161) ~[?:?]
        at net.luckperms.api.query.QueryOptions.contextual(QueryOptions.java:63) ~[?:?]
        at me.lucko.luckperms.common.config.ConfigKeys.lambda$static$1(ConfigKeys.java:108) ~[?:?]
        at me.lucko.luckperms.common.config.ConfigKeyTypes$CustomKey.get(ConfigKeyTypes.java:119) ~[?:?]
        at me.lucko.luckperms.common.config.AbstractConfiguration.load(AbstractConfiguration.java:83) ~[?:?]
        at me.lucko.luckperms.common.config.AbstractConfiguration.<init>(AbstractConfiguration.java:56) ~[?:?]
        at me.lucko.luckperms.common.plugin.AbstractLuckPermsPlugin.enable(AbstractLuckPermsPlugin.java:119) ~[?:?]
        at me.lucko.luckperms.bukkit.LPBukkitBootstrap.onEnable(LPBukkitBootstrap.java:158) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351) [spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:432) [spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:464) [spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:378) [spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:211) [spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:784) [spigot-1.15.2.jar-2020-03-08-1937:git-Spigot-6de3d4b-fc24934]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by Turbotailz** [comment]
I assume this was introduced in this commit https://github.com/lucko/LuckPerms/commit/bb3afa21a41281aa2813e58e2779b1b2727d550c

You'll have to use an earlier build in the meantime.

**Comment by SlimeDog** [comment]
The dev build 1006 is fine. So yes, one of the three commits in build 1007 is the culprit.

**Comment by lucko** [comment]
Thanks for reporting :)

**Comment by SlimeDog** [comment]
Confirmed fix in LuckPerms-Bukkit-5.0.92 (build 1008).
